app/javascript/application.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import "@patternfly/react-core/dist/styles/base.css";
+import { Button } from '@patternfly/react-core';
+
 
 function App() {
-  return (<h1>Hello World!</h1>);
+    return (
+        <div>
+             <Button variant="primary">PF check</Button>
+        </div>
+       
+    );
```

Comment:
I think this can be written as a one liner, e.g:
`const App = () => <Button variant="primary">PF check</Button>`

Reply:
Done